frontend/src/pages/blocks/main/Main.tsx

```diff
@@ -111,9 +134,51 @@ const Main: FunctionComponent = () => {
       )}
       {filteredData.length > 0 && (
         <Table<DecoratedPollingEntity<Block>>
-          columns={columns}
           data={filteredData}
-        ></Table>
+          columns={columns}
+          renderCustomHeader={(headerGroup) => (
+            <Card
+              className={classNames(
+                tableClasses.tableRow,
+                classes.tableRow,
+                tableClasses.headerRow
```

Comment:
Why was it necessary to define a custom UI via `renderCustomRow`/`renderCustomHeader`? Was it so that you could add custom classes to the card (row)?

I think that we should be reusing existing UI as much as possible, otherwise we would need to change multiple component files if we do some changes to the table component - which would be more time consuiming and could lead to some inconsistencies in the UI (e.g. if we forgot to change all the files that define their custom table UI).

Is it possible to not define custom UI and perhaps just use some way to pass down the props that you need (e.g. row class names)?